src/Utilities/overviewRows.js

```diff
@@ -63,14 +65,26 @@ export const rowMapper = (
         'data-position': 'title',
       },
     },
-    version ? `/api/${apiName}` : '',
+    versions ? `/api/${apiName}` : '',
     {
-      title: <Badge>{version}</Badge>,
-      value: version,
+      title: (
+        <Fragment>
+          {versions &&
+            versions.map((version) => (
+              <Link key={version} to={version}>
```

Comment:
I think you might be missing the `apiName` and the leading `/`. this should most likely look like
```jsx
<Link key={version} to={`/${apiName}/${version}`}>...
```

The same goes for the link above

Reply:
Fixed